services/src/main/java/org/fao/geonet/api/records/MetadataSampleApi.java

```diff
@@ -267,6 +267,20 @@ SimpleMetadataProcessingReport addSamples(
                             isTemplate = templateName.startsWith(templateOfSubTemplatePrefix) ?
                                 "t" : "s";
                         }
+                        if (isTemplate.equals("s")) {
+                            // subtemplates loaded here can have a title or uuid attribute
+                            String tryUuid = xml.getAttributeValue("uuid");
+                            if (tryUuid != null && tryUuid.length() > 0) uuid = tryUuid;
+                            title = xml.getAttributeValue("title");
+                            if (title == null || title.length() == 0) {
+                              title = templateName.substring(prefixLength,
+                                templateName.length() - prefixLength);
+                            }
+                            // throw away the uuid and title attributes if present as they
+                            // cause problems for validation
+                            xml.removeAttribute("uuid");
```

Comment:
uuid should not cause validation issue?

Reply:
Indeed, I can't think of a subtemplate situation where the uuid attribute is not allowed, so we will leave it. Also homogeneous with other code.